src/content/docs/logs/ui-data/obfuscation-ui.mdx

```diff
@@ -18,13 +18,11 @@ With log obfuscation rules, you can prevent certain types of information from be
 
 Log obfuscation is available as part of our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices). 
 
-## Overview [#overview]
+## What is log obfuscation? [#overview]
 
-After logs have been shipped to New Relic, sensitive information in the logs can be obfuscated before being stored in [NRDB](/docs/data-apis/get-started/nrdb-horsepower-under-hood/), our New Relic database, by using obfuscation rules.
+Our [log management service](/docs/logs/get-started/get-started-log-management) automatically masks number patterns that we identify as likely being sensitive items, such as credit card numbrers or Social Security numbers. Additionally, with our log obfuscation feature, you can set up additional data privacy for other personally-identifiable information by defining your own custom obfuscation rules right from the log management UI.
```

Comment:
"numbrers" typo but otherwise looks fine. Feel free to merge after your build finishes!